bedrock/mozorg/redirects.py

```diff
@@ -222,6 +222,13 @@ def decider(request, **kwargs):
     redirect(r'^legal/eula/firefox-2/?$', '/about/legal/eula/firefox-2/'),
     redirect(r'^legal/eula/firefox-3/?$', '/about/legal/eula/firefox-3/'),
 
+    # Bug 1209643
+    redirect(r'^legal/bylaws_amendment_1(\.html|/)?', '/foundation/documents/bylaws-amendment-1/'),
```

Comment:
This is fine, but you can use the view name here as well for a bit more future-proofing. (e.g. `foundation.documents.bylaws`, etc.)